### PR TITLE
DDF for Bosch Thermostat II 230V

### PR DESCRIPTION
**Pull request by aSauerwein**
copy/pasted the thermostat2.json and adapter for the wired versions.
changes:

   *  i removed the config/battery
   *  added state/valve with read&parse of 0x4020

open questions

  * I did not understand what the "bindings" are used for
  deconz-gui still shows the battery icon. don't know what needs to be changed to get that icon removed.
  * for state/valve I just used Item.val = Attr.val but I'm not sure if that correctly matches the percent valve opening
  * I cannot control the thermostat with phoscon web app. It just displays 40F and nothing changes, but it's working fine over homeassistant and rest api

fixes https://github.com/dresden-elektronik/deconz-rest-plugin/issues/7553

### CODE REVIEW

**Comment by mattreim** [comment]
Hello aSauerwein,

with this changes should be function the DDF:

1. Correction file name: "room_thermostat2_230V"
2. Add product name: "Room thermostat II 230V (BTH-RM230Z)"
3. Delete: Battery
4. Add: valve

[room_thermostat2_230V.json](https://github.com/dresden-elektronik/deconz-rest-plugin/files/14193597/room_thermostat2_230V.json)

**Comment by aSauerwein** [comment]
hi @mattreim 
thanks! i imported your .json and it looks fine!

**Comment by mattreim** [comment]
@aSauerwein check whether the device supports this cluster.

poll_control_cluster

File: devices/bosch/room_thermostat2_230V.json [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+        "$address.ext",
+        "0x01",
+        "0x0201"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0301",
+        "endpoint": "0x01",
+        "in": [
+          "0x0001",
```
Comment (SwoopX) [inline devices/bosch/room_thermostat2_230V.json:23]: Please remove entry `0x0001` here, we do not have a battery powered device this time 😉

File: devices/bosch/room_thermostat2_230V.json [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+        "$address.ext",
+        "0x01",
+        "0x0405"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0301",
+        "endpoint": "0x01",
+        "in": [
+          "0x0001",
```
Comment (SwoopX) [inline devices/bosch/room_thermostat2_230V.json:295]: Please remove entry `0x0001` here, we do not have a battery powered device this time 😉

File: devices/bosch/room_thermostat2_230V.json [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/checkin",
+          "public": false,
+          "refresh.interval": 3600,
+          "default": 14400
+        },
```
Comment (SwoopX) [inline devices/bosch/room_thermostat2_230V.json:85]: As it has already been mentioned, this part should be removed, as it wouldn't work out based on the provided screenshot.

**Review by SwoopX — CHANGES_REQUESTED** [review]
Just a few minors to catch up with 👍

**Review by SwoopX — APPROVED** [review]
Should be good to go now, thanks!

**Pull request merged**